0_RootFS/gcc_common.jl

```diff
@@ -211,14 +208,55 @@ function gcc_script(compiler_target::Platform)
         make install
 
         # Install cctools
+        cd ${WORKSPACE}/srcdir/cctools-port/cctools
+        ./autogen.sh
         mkdir -p ${WORKSPACE}/srcdir/cctools_build
         cd ${WORKSPACE}/srcdir/cctools_build
-        CC=/usr/bin/clang CXX=/usr/bin/clang++ LDFLAGS=-L/usr/lib ${WORKSPACE}/srcdir/cctools-port/cctools/configure \
+        # CC=/usr/bin/clang CXX=/usr/bin/clang++ LDFLAGS=-L/usr/lib ${WORKSPACE}/srcdir/cctools-port/cctools/configure \
+        #     --prefix=${prefix} \
+        #     --target=${COMPILER_TARGET} \
+        #     --host=${MACHTYPE} \
+        #     --with-libtapi=${prefix}
+
+        # Does not work:
+        #     CFLAGS="-isysroot ${sysroot}"
+        #     CXXFLAGS="-isysroot ${sysroot}"
+        #     CPPFLAGS="-isystem ${sysroot}/usr/include"
+        #     CPPFLAGS="-isystem /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/include"
+        #     CPPFLAGS="-isystem /workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers"
+
+        # $sysroot:
+        #     /workspace/destdir/x86_64-apple-darwin14/sys-root
+        # find:
+        #     /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/include/sys/cdefs.h
+        #     /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+        #     /workspace/srcdir/cctools-port/cctools/include/sys/cdefs.h
+        #     /workspace/srcdir/gcc-14.2.0/fixincludes/tests/base/sys/cdefs.h
+        #     /workspace/srcdir/MacOSX10.12.sdk/usr/include/sys/cdefs.h
+        #     /workspace/srcdir/MacOSX10.12.sdk/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+        #     /workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/usr/include/sys/cdefs.h
+        #     /workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+
+        # find / -name cdefs.h >CDEFS-LOCS
```

Comment:
The missing `sys/cdefs.h` is a bug in cctools: https://github.com/tpoechtrager/cctools-port/issues/164
The solution isn't [using target header files](https://github.com/JuliaPackaging/Yggdrasil/pull/10132#discussion_r1901917587), but to provide a usable `sys/cdefs.h`, e.g. https://github.com/openembedded/openembedded-core/blob/master/meta/recipes-core/musl/bsd-headers/sys-cdefs.h

Reply:
Congratulations. That's probably why I am stuck.